app/components/article-featured-video.js

```diff
@@ -92,6 +95,17 @@ export default Component.extend(
 				});
 			});
 
+			// when playing video on article with infobox, closing fullscreen also has to pause video
+			// as it will be not visible
+			player.mb.subscribe(window.OO.EVENTS.FULLSCREEN_CHANGED, 'ui-display-update', (name, isFullScreen, paused) => {
+				this.set('isPlaying', isFullScreen);
```

Comment:
we change `isPlaying` event if it's article without infobox? And it's confusing assigning isFullScreen to isPlaying